src/familiar/itemFamiliar.ts

```diff
@@ -0,0 +1,66 @@
+import { Familiar, myFamiliar, runChoice, useFamiliar, visitUrl } from "kolmafia";
+import { $familiar, $item, findFairyMultiplier, get, have, set } from "libram";
+import { menu } from "./freeFightFamiliar";
+
+let bestNonCheerleaderFairy: Familiar;
+
+export function bestFairy(): Familiar {
+  if (have($familiar`Trick-or-Treating Tot`) && have($item`li'l ninja costume`)) {
+    return $familiar`Trick-or-Treating Tot`;
+  }
```

Comment:
If we're expecting our item familiars to be at least 70lbs, a Jumpsuited Hounddog matches the ninja tot at 67.8lbs (while regular fairies match it at 84.7lbs). I'm not sure if we want an early return here if we're running with that assumption.

Reply:
This is duplicating existing functionality; I do think that's a reasonable shout, especially because a decked out account is hitting something like 50 lbs without breaking a sweat, but it's hard to decide what exactly is optimal. We could start using `+switch` in our maximizer strings, but that would be problematic with the way outfit caching works

Reply:
Could we not first choose the best fairy, then calculate if that best fairy would beat out a ninja tot? 
I suppose that would require outfit caching which is not done here; but a clear cut case would be one where our best fairy beats out the ninja tot pre-maximizer (i.e. in our current outfit), then clearly we shouldn't be using the tot over that other option.

Reply:
I think using `+switch` in our maximizer string is probably the easist way to achieve that

Reply:
This is what current behavior is, so I'm going to merge as-is, but it's something we can address in a little bit. I just will never merge it if I keep widening the scope of this PR.